CredentialCardList.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="s-collapsible" @click="toggleFields()">
+    <div class="row bg-white s-card">
+      <div class="row justify-between items-center s-card-info-top">
+        <div class="s-card-info-top-text row">
+          <q-icon :name="toggleArrow" class="s-arrow" />
+          <q-item multiline>
+            <q-item-main class="s-card-info-top-text-lines">
+              <q-item-tile label class="s-card-title" lines="2">
+                {{credential.name}}
+              </q-item-tile>
+              <q-item-tile
+                sublabel lines="2"
+                class="s-sublabel">
+                {{credential.description}}
+              </q-item-tile>
+            </q-item-main>
+          </q-item>
+        </div>
+        <q-card-media>
+          <q-icon
+            v-if="useDefaultImage || !credential.issuerLogo"
+            :name="defaultImage" />
+          <img
+            v-else
+            :src="credential.issuerLogo"
+            @error="imageError">
+        </q-card-media>
+      </div>
+      <q-card-separator />
+    </div>
+
+    <div v-if="showFieldValues === true">
```

Comment:
Can this just test for truthiness?
```suggestion
    <div v-if="showFieldValues">
```